File: code-coaches/medium/initials/initials.py
def get_initials(name: str) -> str:
    name_split = name.split()
    return f"{name_split[0][0]}{name_split[1][0]}"


def main():
    num_names = int(input())
    names = [get_initials(input()) for _ in range(num_names)]
    print(" ".join(names))


if __name__ == "__main__":
    main()

File: code-coaches/medium/initials/Initials.java
import java.util.Scanner;

public class Initials {

    public static String getInitials(String name) {
        String[] nameSplit = name.split(" ");
        return nameSplit[0].charAt(0) + "" + nameSplit[1].charAt(0);
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int numNames = scanner.nextInt();
        scanner.nextLine(); // Consume the newline character after the integer input

        String[] names = new String[numNames];
        for (int i = 0; i < numNames; i++) {
            names[i] = getInitials(scanner.nextLine());
        }

        System.out.println(String.join(" ", names));

        scanner.close();
    }

}